src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -823,6 +829,25 @@ $kCodeBlockDarkLineMatchBackgroundColor: rgba(255, 255, 255, 0.12); // we don't
     padding: var(--kui-space-40, $kui-space-40);
     position: relative;
 
+    @media (min-width: $kui-breakpoint-laptop) {
+      &:hover {
+        .code-block-secondary-actions {
+          opacity: 1;
+
+          :deep {
+            .code-block-copy-button {
+              opacity: 1;
+            }
+          }
+
+          .secondary-actions {
+            opacity: 1;
+          }
+        }
+
+      }
+    }
```

Comment:
I think it's better to move this block to where we style `.secondary-actions` below. That makes the styling logic clearer.

A simplified example: https://sass-lang.com/playground/#eJxtzs8KgzAMBvB7nyKHMbZDxO406mWvUtuCodoUjdtk+O77g9s8mEMCH+HHp/UujoR1H2zMTEmwtVk4G9Cnc5nvlVKFYx+wbtlFHILj5G0/oXVCnAZ4KIBLFzxZOHSU8EZeGgPb6vHzDlBsO+/hbB3JZKCslmQ5RcR/EViXelESkpiGr6GH/Y9aYXqJZvXds5rVE6bkTKU=